next.config.js

```diff
@@ -1,19 +1,19 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const optimizedImages = require('next-optimized-images')
-
-/** @type {import('next').NextConfig} */
-const nextConfig = optimizedImages({
-  reactStrictMode: true,
-  handleImages: ['svg'],
-  webpack: (config, { isServer }) => {
-    if (!isServer) {
-      // Fixes npm packages that depend on `fs` module
-      // https://github.com/vercel/next.js/issues/7755#issuecomment-937721514
-      config.resolve.fallback.fs = false
-    }
-    config.resolve.mainFields = ['browser', 'main', 'module']
-    return config
-  },
-})
-
-module.exports = nextConfig
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
+  images: {
+    loader: 'akamai',

```

Comment:
What's this about an Akamai loader? Why do we need any sort of third party service involved with our image loading?

Reply:
This is I think built-in next and it is just there to compress images

https://nextjs.org/docs/api-reference/next/legacy/image#built-in-loaders

Added this as we used a loader before to optimize images as next does not optimize svg I believe